Source/WTF/wtf/text/StringCommon.cpp

```diff
@@ -83,7 +83,7 @@ const uint16_t* find16AlignedImpl(const uint16_t* pointer, uint16_t character, s
         oldLength = length;
         length -= stride;
         cursor += stride;
-    } while (length < oldLength);
+    } while (length && length < oldLength);
```

Comment:
It looks like the purpose of the `length < oldLength` condition is to detect an underflow.  The only way that is possible is if `length` is less than stride.  Why not change this condition to `length >= stride`?

Reply:
We can also remove `oldLength` since it is not needed anymore.

Reply:
Fixed.